### ISSUE TITLE
Instagram proofs update

### ISSUE THREAD

**Issue by yknl** [stx-labs/stacks.js#417]
Update Instagram social proof checking as Instagram has changed the format of the description meta tag from

```
<meta content="2 Likes, 1 Comments - username (@username) on Instagram: “Verifying my Blockstack ID is secured with the address 1KlNv27j3kk1YfbyykdJ8H3npRejs8d...”" name="description" />
```

to

```
<meta content="2 Likes, 1 Comments - @username on Instagram: “Verifying my Blockstack ID is secured with the address 1KlNv27j3kk1YfbyykdJ8H3npRejs8d…”" name="description" />
```

The required code changes are here:
https://github.com/blockstack/blockstack.js/blob/master/src/profiles/services/instagram.js#L42
and possibly
https://github.com/blockstack/blockstack.js/blob/master/src/profiles/services/instagram.js#L54

Test case needs an update as well:
https://github.com/blockstack/blockstack.js/blob/master/tests/testData/profiles/ken.verification.instagram.html#L57

**Comment by kantai** [comment]
A fix was merged into develop (thanks @cponeill!)

**Comment by cponeill** [comment]
Glad I could help out.

**Comment by kantai** [comment]
This was merged into master in release 17.2.0
